### DISCUSSION TITLE
Wrong Hour

### DISCUSSION THREAD

**Discussion by abachtold** [kimai/kimai#3040] [Questions & Answers]
I think that it has some kind of configuration with "Summer Time" that it takes an hour from the task that I do on it. In the task the time is 09 to 18. In the timesheet it's 08-17. Where do I configure to stop this behavior?

**Comment by kevinpapst** [comment]
[It is about your timezone. Check your user settings.](https://www.kimai.org/documentation/user-preferences.html#timezone)

**Comment by abachtold** [comment]
My timezone is correct. But the savetime isn't active this year, so I think that is generating the problem. How can I disable the auto savetime?

**Reply by kevinpapst** [reply]
You need to update the libraries in your operating system. Kimai does not manage these informations, they are pulled in through PHP.
